### PR TITLE
OCPCLOUD-1812: Update tooling & k8s 1.26

### PR DESCRIPTION
**Pull request by RadekManak**
Updates test tooling to be vendored instead of using shell script to download it. Also updates k8s to 1.26.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/openshift/release:golang-1.18 AS builder
+FROM registry.ci.openshift.org/openshift/release:golang-1.19 AS builder
```
Comment (lobziik) [inline Dockerfile:1]: I wondering why the ART bot didn't update this?

**Review by lobziik — COMMENTED** [review]
I'm a little bit curious about why go version was not updated by ART automation, but don't think that it's a reason for blocking this

/approve

File: Dockerfile (hunk 1 above)
Comment (RadekManak) [reply Dockerfile:1]: ART builds use Dockerfile.rhel. Not sure if this Dockerfile is used anywhere right now.

**Comment by JoelSpeed** [comment]
/retest-required

Looks like the unit tests are suddenly taking longer to run and hitting a global timeout 🤔

**Pull request merged**